crates/stages/src/stages/hashing_storage.rs

```diff
@@ -116,8 +116,18 @@ impl<DB: Database> Stage<DB> for StorageHashingStage {
 
             collect(&mut channels, &mut collector)?;
 
+            let total_hashes = collector.len();
+            let interval = (total_hashes / 10).max(1);
             let mut cursor = tx.cursor_dup_write::<tables::HashedStorages>()?;
-            for item in collector.iter()? {
+            for (index, item) in collector.iter()?.enumerate() {
+                if index > 0 && index % interval == 0 {
+                    info!(
+                        target: "sync::stages::hashing_storage",
+                        progress = format!("{:.2}%", (index as f64 / total_hashes as f64) * 100.0),
+                        "Inserting hashes"
+                    );
+                }
+
```

Comment:
I see that storage hashing takes 50min on a beefy machine (https://github.com/paradigmxyz/reth/pull/7030), so we'll log the progress every 5 minutes or so. But if you have worse hardware, I guess it'll take longer and the progress will also be reported infrequently.

I think we should make it time-dependent, so eg log the progress every 1 minute or 5 minutes

Reply:
on a second thought, doesn't make sense to do it time based here, since insertion is way faster than collection phase.

WIth the current settings we're logging at every 1M proceed entries, which may be fine?